test/yup-middleware.spec.ts

```diff
@@ -127,6 +127,52 @@ describe('express yup middleware', () => {
     })
   })
 
+  describe('using yup validation options', () => {
```

Comment:
Since you're testing specifically "abortEarly" option I'd have that indicated in the describe text.
On the other hand if you plan to have other settings tested, abortEarly:false could be set in the "it" test, and the schema would be built with the default options.

Reply:
I've created a new layer of description for the specific property being tested.